dbms/src/Flash/Mpp/tests/gtest_mpptunnel.cpp

```diff
@@ -27,6 +27,16 @@ namespace DB
 {
 namespace tests
 {
+namespace
+{
+TrackedMppDataPacketPtr newDataPacket(const String & data)
+{
+    auto data_packet_ptr = std::make_shared<TrackedMppDataPacket>();
+    data_packet_ptr->getPacket().set_data(data);
```

Comment:
Set data is not enough, most of the information is stored in chunks. Why not just copy the whole packet?

Reply:
Because the original ut only set data here